app/src/main/res/layout/app_version_activity.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context=".administratorcontrols.appversion.AppVersionActivity">
+  <com.google.android.material.appbar.AppBarLayout
+    android:id="@+id/app_version_app_bar_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:theme="@style/ThemeOverlay.AppCompat.ActionBar"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent">
+    <androidx.appcompat.widget.Toolbar
+      android:id="@+id/app_version_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="?attr/actionBarSize"
+      android:background="@color/oppiaPrimaryDark"
+      android:textSize="20sp"
+      app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
+      app:titleTextColor="@color/white"/>
+  </com.google.android.material.appbar.AppBarLayout>
+  <FrameLayout
+    android:id="@+id/app_version_fragment_placeholder"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    app:layout_constraintVertical_weight="1"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/app_version_app_bar_layout"/>
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done